#!/usr/bin/env python
PACKAGE = "openpose_ros" # package name

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# name , type , level ,description , default ,min ,max
gen.add("key_point_threshold", double_t, 0, "Threshold of heat map for key points", 0.15, 0., 1.)
gen.add("affinity_threshold", double_t, 0, "Threshold of part affinity field", 0.03, 0., 1.)
gen.add("line_division", int_t, 0, "Number of samples within line segments", 15, 2, 100)

exit(gen.generate(PACKAGE, "openpose", "KeyPointDetector"))
